google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java

```diff
@@ -185,17 +185,24 @@ private void runExample(
       long customerId,
       String keyword,
       String freeFormKeywordText,
-      Long locationId,
+      String locationId,
       String businessName) {
     if (locationId == null && businessName == null) {
       throw new RuntimeException("Must provider either --locationId or --businessName");
     }
+
+    // [START smart_campaign_14]
```

Comment:
Can you change this to `add_smart_campaign_14`

Reply:
Done